specs/agents/metadata.md

```diff
@@ -22,6 +22,18 @@ System metadata relates to the host/container in which the service being monitor
    - pod name
    - pod UID
 
+#### Hostname
+
+The `system.hostname` field is deprecated in favour of two newer fields:
```

Comment:
It would be super helpful if such deprecation notes would provide information on which APM Server version the behavior change is implemented (maybe add a placeholder if it is not clear yet).

In this case, it looks like the fields were added in 7.4.0.

Reply:
Good call, I'll add this info, thanks!